resource/lvm/lv/lv.go

```diff
@@ -0,0 +1,133 @@
+package lv
+
+import (
+	"fmt"
+	"github.com/asteris-llc/converge/load/registry"
+	"github.com/asteris-llc/converge/resource"
+	"github.com/asteris-llc/converge/resource/lvm/lowlevel"
+	"github.com/asteris-llc/converge/resource/wait"
+)
+
+type ResourceLV struct {
+	group      string
+	name       string
+	size       int64
+	sizeOption string
+	sizeUnit   string
+	lvm        lowlevel.LVM
+	needCreate bool
+	devicePath string
+}
+
+type Status struct {
+	resource.Status
+	DevicePath string
+}
+
+func (r *ResourceLV) Check(resource.Renderer) (resource.TaskStatus, error) {
+	status := &Status{}
+	ok, err := r.checkVG()
+	if err != nil {
+		return nil, err
```

Comment:
Probably a good idea to wrap the errors with "check", "apply", etc. to give the user more information.